ui/pages/settings/snaps/view-snap/view-snap.js

```diff
@@ -66,6 +66,8 @@ function ViewSnap() {
   const [isShowingRemoveWarning, setIsShowingRemoveWarning] = useState(false);
   const [isDescriptionOpen, setIsDescriptionOpen] = useState(false);
   const [isOverflowing, setIsOverflowing] = useState(false);
+  // eslint-disable-next-line no-unused-vars -- Main build does not use setIsRemovingKeyringSnap
+  const [isRemovingKeyringSnap, setIsRemovingKeyringSnap] = useState(false);
```

Comment:
Should this be code fenced with `keyring-snaps`? I'm assuming this is a problem in the CI

Reply:
We use the value `isRemovingKeyringSnap` but not `setIsRemovingKeyringSnap` in the main build

Reply:
Resolved in a sync